### PR TITLE
Updated danish GUI translation

### PR DESCRIPTION
**Pull request by cryptobench**
Sorry for the delay. :-)

### CODE REVIEW

File: translations/monero-core_da.ts [hunk 1]
```diff
@@ -1,2508 +1,5023 @@
...
-    </message>
-    <message>
-        <location filename="../pages/Transfer.qml" line="538"/>
-        <source>Can&apos;t load unsigned transaction: </source>
-        <translation>Kan ikke loade usignerede transaktioner: </translation>
-    </message>
-    <message>
-        <location filename="../pages/Transfer.qml" line="545"/>
-        <source>
-Number of transac
```
Comment (erciccione) [inline translations/monero-core_da.ts]: Something happened from this line, looks like the file is duplicated

**Comment by ordtrogen** [comment]
Here's something you might want to look at: 
There are two different translations of "unlocked balance". I'd say go for "Oplåst saldo":

image

image

**Comment by cryptobench** [comment]
@ordtrogen You're right. I did all of this in school a couple of hours ago. Might have gotten a little distracted and made a mistake. Will fix now

File: translations/monero-core_da.ts (hunk 1 above)
Comment (cryptobench) [reply translations/monero-core_da.ts]: I'm a little unsure what the problem is here

Comment (erciccione) [inline translations/monero-core_da.ts]: from this point on the file is duplicated of the precedent strings, please fix

Comment (erciccione) [reply translations/monero-core_da.ts]: don't know why GitHub is showing the wrong string (again), the file is duplicated from line 2515

**Comment by cryptobench** [comment]
@erciccione I have no idea how it duplicated, wtf. Seems to have happened when I uploaded.
I have deleted the duplicate now though.

File: translations/monero-core_da.ts [hunk 2]
```diff
@@ -1,2508 +1,2514 @@
...
```
Comment (erciccione) [inline translations/monero-core_da.ts]: Please add `:`

Comment (erciccione) [inline translations/monero-core_da.ts]: Please add `:`

Comment (erciccione) [inline translations/monero-core_da.ts]: Why the question mark?

Comment (cryptobench) [reply translations/monero-core_da.ts]: I'm not sure which one we're talking about. Github is bugging again :/

Comment (cryptobench) [reply translations/monero-core_da.ts]: I'm not sure which one we're talking about. Github is bugging again :/

Comment (cryptobench) [reply translations/monero-core_da.ts]: I'm not sure which one we're talking about. Github is bugging again :/

**Comment by erciccione** [comment]
GitHub clearly hate this PR 
I was referring to lines 1122 1653 1849

**Comment by cryptobench** [comment]
@erciccione added missing characters and removed the question mark. Don't ask me why I added a question mark... haha :laughing:

**Comment by erciccione** [comment]
Weird things happened here lol. Btw, squash those commits please so it can get merged

**Comment by cryptobench** [comment]
Gimme some time. My old brain has to remember how I did it the last time. :p

**Comment by ordtrogen** [comment]
A number of places:

Antal -> beløb
?

Is password "adgangskode" or "kodeord" in Danish?

Line 1611:
Kan ikke loade ... -> Kan ikke indlæs ...
?

Line 1913:
transaktions gebyret -> transaktionsgebyret
?

Line 2155:
ord -> bogstav
?

Line 2223:
slado -> saldo



2018-03-07 20:46 GMT+01:00 Phillip Jensen <notifications@github.com>:

**Comment by erciccione** [comment]
This still need to be squashed, let me know if you need help @cryptobench

**Comment by erciccione** [comment]
@cryptobench we really need to squash those commits before the code freeze

**Comment by erciccione** [comment]
@cryptobench Contact me as soon as you can please, we really need to get this PR ready

**Comment by erciccione** [comment]
@cryptobench @luigi1111 this can be closed

**Comment by skftn** [comment]
+resolved

**Pull request closed without merging**